src/main/scala/mesosphere/marathon/core/launchqueue/LaunchQueue.scala

```diff
@@ -69,6 +69,7 @@ trait LaunchQueue {
 
   /** Remove all instance launch requests for the given PathId from this queue. */
   def purge(specId: PathId): Unit
```

Comment:
That's a good point. We've discussed it for the review on master and declared it as deprecated and will remove it in 1.6. What do you think?